src/System.AppContext/src/project.json

```diff
@@ -0,0 +1,12 @@
+{
+  "dependencies": {
+    "Microsoft.TargetingPack.Private.NetNative": "1.0.0-rc2-23530",
```

Comment:
Yes this is the .NET Native version but I've just asked @dsgouda to add the other versions as well.